app/tests/cases_tests/factories.py

```diff
@@ -152,3 +152,8 @@ def files(self, create, extracted, **kwargs):
         if create and not extracted:
             ImageFileFactoryWithMHDFile3DLarge4Slices(image=self)
             ImageFileFactoryWithRAWFile3DLarge4Slices(image=self)
+
+
+class RawImageUploadSessionFactory(factory.DjangoModelFactory):
```

Comment:
The subfactory for algorithm (images) is missing

Reply:
Sorry I dont understand.

Reply:
Ok, in this case it doesn't matter a huge amount, we can add it later